### DISCUSSION TITLE
[3.x] Model props, also fills appended attributes

### DISCUSSION THREAD

**Discussion by megasteve19** [livewire/livewire#6121] [7. Bugs (archived)]
### Livewire version

3.0@beta

### Which PHP version are you using?

PHP 8.2

### Steps To Reproduce

1. Create a model with appended attribute
2. Add this model as property in component
3. Add an input and add `wire:model`
4. On updating hook update model `$this->model->update($this->validate())` with validated input

You will see that it also contains appended attributes so sql will throw error that saying there's no such a column.

### Additional information

I'm not a Livewire expert, I was waiting v3 to give a shot. So I can only guess what's the problem is. My guess is Livewire gets all the the attributes from client than puts into model without checking it. And seems like Laravel doesn't care if you fill an unfillable attribute.

Currently I'm bypassing this problem by just refreshing the model:
```php
$this->post->refresh();
$this->post->update($validated);
```

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by joshhanley** [comment]
@megasteve19 thanks for reporting! Can you please add an example Livewire component class and view with screenshots that demonstrates the issue so we can investigate? Thanks!

**Comment by megasteve19** [comment]
@joshhanley I'm very sorry, it's my mistake. Seems like I'm the who's setting attribute. Sorry, old code new problem. I was creating sample project for this "bug" but seems like everything works pretty well. So there's no problem with Livewire. Thank you.
